pgo-scheduler/scheduler/types.go

```diff
@@ -18,14 +18,11 @@ package scheduler
 import (
 	"time"
 
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-
+	"github.com/crunchydata/postgres-operator/internal/kubeapi"
 	cv2 "github.com/robfig/cron"
 )
 
-var kubeClient kubernetes.Interface
-var restClient *rest.RESTClient
+var clientset kubeapi.Interface
```

Comment:
This is another change to the clients in scheduler. It is initialized in `scheduler.go`.